e2e/schema_registry/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+docker-compose up -d zookeeper
+docker-compose up -d kafka kafka
+docker-compose up -d mysql mysql
+docker-compose up -d connect connect
+docker-compose up -d schema-registry schema-registry
+
+docker-compose up -d spark-master spark-master
+docker-compose up -d spark-worker spark-worker
+docker-compose up --exit-code-from spark-submit spark-submit
+docker-compose up --exit-code-from schema-registry-register schema-registry-register
+docker-compose up --exit-code-from mysql-client mysql-client
+docker-compose up --exit-code-from kafka-consume-avro kafka-consume-avro
+exit_code=$(docker ps -aq -f label=com.docker.compose.project=kafka-consume-avro | xargs -I{} docker inspect {} --format='{{.State.ExitCode}}' | paste -sd+ - | bc)
```

Comment:
kafka-consume-avro is the name of the project?

Reply:
I actually don't think you need this at all

Reply:
it is the kafka consumer that consumes the result and compares to expected